clrenv/read.py

```diff
@@ -108,7 +108,7 @@ def read(self) -> NestedMapping:
 
         return result
 
-    def postprocess_str(self, value: str) -> str:
+    def postprocess_str(self, value: str) -> Union[str, Secret]:
```

Comment:
```suggestion
        def postprocess_str(self, value: str) -> Union[str, Secret]:
        """Post process string values."""
        # Expand environmental variables in the form of $FOO or ${FOO}.
        value = os.path.expandvars(value)

        # If value is a path starting with ~, expand.
        if value.startswith("~"):
            return os.path.expanduser(value)
        # Substitute from clrypt keyfile.
        if value.startswith("^keyfile "):
            return Secret(source=value, value=self.evaluate_clrypt_key(value[9:]))
        # Substitute from aws ssm parameter store.
        if value.startswith("^parameter "):
            return Secret(source=value, value=self.evaluate_ssm_parameter(value[11:]))

        return value
```

This will fix the mypy error

Reply:
```
(venv_clrenv) mpcusack@mpcp15:~/dev/clrenv$ mypy clrenv
clrenv/read.py:120: error: Incompatible types in assignment (expression has type "Secret", variable has type "str")
clrenv/read.py:123: error: Incompatible types in assignment (expression has type "Secret", variable has type "str")

```